### PR TITLE
power_carried_by_coaxial_waveguide

### PR DESCRIPTION
**Pull request by Zulusssss**
Add law and tests.

### CODE REVIEW

File: symplyphysics/laws/electricity/circuits/power_carried_by_coaxial_waveguide.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+waveguide_power = Symbol("waveguide_power", units.power)
+
+relative_permittivity = Symbol("relative_permittivity", dimensionless)
+relative_permeability = Symbol("relative_permeability", dimensionless)
+voltage = Symbol("voltage", units.voltage)
+outer_diameter = Symbol("outer_diameter", units.length)
+inner_diameter = Symbol("inner_diameter", units.length)
+
+resistance = Quantity(120 * units.ohm)
```
Comment (blackyblack) [inline symplyphysics/laws/electricity/circuits/power_carried_by_coaxial_waveguide.py:26]: `vacuum_impedance` ?

File: symplyphysics/laws/electricity/circuits/power_carried_by_coaxial_waveguide.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    return print_expression(law)
+
+
+@validate_input(relative_permittivity_=relative_permittivity,
+    relative_permeability_=relative_permeability,
+    voltage_=voltage,
+    outer_diameter_=outer_diameter,
+    inner_diameter_=inner_diameter)
+@validate_output(waveguide_power)
+def calculate_waveguide_power(relative_permittivity_: float, relative_permeability_: Quantity,
```
Comment (blackyblack) [inline symplyphysics/laws/electricity/circuits/power_carried_by_coaxial_waveguide.py:41]:
```suggestion
def calculate_waveguide_power(relative_permittivity_: float, relative_permeability_: float,
```

File: symplyphysics/laws/electricity/circuits/power_carried_by_coaxial_waveguide.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    validate_output, dimensionless,)
+
+
+# Description
+## A coaxial waveguide is an electrical cable consisting of a central conductor and a shield arranged coaxially and separated
+## by an insulating material or an air gap. It is used to transmit radio frequency electrical signals.
+
+## Law is: P = (U^2 / 120) *  sqrt(er / (mur * ln(D / d))), where
+## P - power carried by the waveguide,
+## U - voltage,
```
Comment (blackyblack) [inline symplyphysics/laws/electricity/circuits/power_carried_by_coaxial_waveguide.py:12]: Voltage between what points?

Comment (Zulusssss) [reply symplyphysics/laws/electricity/circuits/power_carried_by_coaxial_waveguide.py:12]: > Voltage between what points?

Between the ground and the cable.

**Review by blackyblack — APPROVED** [review]

**Pull request merged**